experiments/Train.ipynb

```diff
@@ -159,380 +150,840 @@
+   "id": "eb92b58e",
+   "metadata": {},
+   "source": [
+    "# Build model"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
-   "id": "ac13d56e",
+   "id": "2ee58091",
    "metadata": {},
    "outputs": [],
    "source": [
-    "training_ds = create_dataframe(\"train\", \"xx\")\n",
-    "test_ds = create_dataframe(\"test\", \"xx\")\n",
-    "val_ds = create_dataframe(\"val\", \"xx\")"
+    "tf.random.set_seed(42)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "b1bfc7ed",
+   "id": "993bb1d8",
    "metadata": {},
    "source": [
-    "# Configuration"
+    "## Model inputs"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 6,
-   "id": "a92c3dd7",
-   "metadata": {
-    "scrolled": true
-   },
+   "id": "531185de",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "# load config json\n",
-    "import json\n",
-    " \n",
-    "# Opening JSON file\n",
-    "with open('../config.json') as json_file:\n",
-    "    json_config = json.load(json_file)\n",
-    "json_config \n",
+    "# we use dicts so rerunning individual model cells is idempotent\n",
+    "inputs = {}\n",
+    "input_graphs = {}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "8353a5f1",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-05-18 01:05:58.379423: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory\n",
+      "2022-05-18 01:05:58.379447: W tensorflow/stream_executor/cuda/cuda_driver.cc:269] failed call to cuInit: UNKNOWN ERROR (303)\n",
+      "2022-05-18 01:05:58.379459: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (alba): /proc/driver/nvidia/version does not exist\n",
+      "2022-05-18 01:05:58.379641: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F FMA\n",
+      "To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.\n"
+     ]
+    }
+   ],
+   "source": [
+    "ds = tfds.load('off_categories', split=TRAIN_SPLIT)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "c61fec40",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 25.8 s, sys: 11.9 s, total: 37.6 s\n",
+      "Wall time: 6.49 s\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "93000"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "\n",
+    "feature_name = 'product_name'\n",
```

Comment:
It would be cool to comment more this kind of cells in the future. What are we doing, why we try this, etc.

Reply:
I'll pass for this PR in the interest of time, but good point. If someone wants to tackle that part, please go ahead!!